salt/modules/pf.py

```diff
@@ -0,0 +1,349 @@
+# -*- coding: utf-8 -*-
+'''
+Control the OpenBSD packet filter (PF).
+
+:codeauthor: Jasper Lievisse Adriaanse <j@jasper.la>
+
+.. versionadded:: Fluorine
+'''
+
+from __future__ import absolute_import
```

Comment:
For Py2/Py3 compatibility, we need this line to be:
```
from __future__ import absolute_import, print_function, unicode_literals
```

Reply:
Sure thing; I've just pushed the fix.